### ISSUE TITLE
ECS Agent Not Starting When IPv6 is Disabled

### ISSUE THREAD

**Issue by aristenn** [aws/amazon-ecs-init#323]
### Summary
ECS Agent Not Starting When IPv6 is Disabled

### Description
We are running a customized AMI built on top of the ECS Optimized Amazon Linux 2 Image. We apply CIS Benchmarks that then disable IPv6.

### Expected Behavior
We expect the ECS Agent to start

### Observed Behavior
The ECS agent is showing as stopped:
```
7b48a1c1c376        amazon/amazon-ecs-agent:latest   "/agent"            20 hours ago        Exited (143) 20 hours ago                       ecs-agent
```

None of our tasks are able to start due to the agent being stopped

### Supporting Log Snippets
We are receiving the following error repeatedly in /var/log/ecs/ecs-init-log:
```
2020-04-02T17:41:27Z [INFO] pre-start
2020-04-02T17:41:27Z [ERROR] Error disable ipv6 router advertisements exit status 255; raw output: sysctl: cannot stat /proc/sys/net/ipv6/conf/docker0/accept_ra: No such file or directory

2020-04-02T17:41:27Z [ERROR] could not disable ipv6 router advertisements: exit status 255
```

**Comment by aristenn** [comment]
As an additional data point, I've removed the line disabling IPv6 and built a new AMI. The ECS agent started up and my tasks are running, and I can hit my application front end.

**Comment by yunhee-l** [comment]
Thank you for reporting this issue - can you please provide what version of Agent (or ECS Optimize AMI) you are seeing this issue in?

**Comment by aristenn** [comment]
We are using Agent Version 1.38.0

**Comment by yunhee-l** [comment]
Thank you, we are looking into this issue with high priority.

**Comment by sparrc** [comment]
Hello @aristenn, could you let us know what steps you are taking to disable ipv6? Could you also let us know if you are setting up any sort of user-defined docker bridge network?

ie, are you doing something that might disable the default 'docker0' bridge network?

**Comment by aristenn** [comment]
The `docker0` network is enabled, I don't believe that we're doing anything custom with it.

When running our AMI build here are the relevant lines where we disable ipv6:
```
sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT="/GRUB_CMDLINE_LINUX_DEFAULT="ipv6.disable=1 /' /etc/default/grub
grub2-mkconfig -o /boot/grub2/grub.cfg
```

**Comment by sparrc** [comment]
@aristenn thanks for the steps to reproduce, we have a PR up to fix this issue: https://github.com/aws/amazon-ecs-init/pull/324

**Comment by ubhattacharjya** [comment]
Hi @aristenn 
The fix has been published in agent version 1.39.0. Kindly update the agent.

Closing this issue. Please reopen if the issue persists.

**Comment by yrral86** [comment]
This issue seems to be back with a slightly different file
```
systctl: cannot stat /proc/sys/net/ipv6/conf/all/accept_redirects: no such or directory
```
